tests/plugins/test_deepspeed_plugin.py

```diff
@@ -265,6 +266,10 @@ def backward(self, loss: Tensor, optimizer: Optimizer, optimizer_idx: int, *args
                                                     (RandomIterableDataset, "auto"), (RandomIterableDataset, 10)])
 def test_deepspeed_auto_batch_size_config_select(tmpdir, dataset_cls, value):
```

Comment:
I bet this would fail if we set gpus > 1 due to the parameterization :) haha

Reply:
That could explain why some work with `gc.collect` and others do not

Reply:
yep! let's check tomorrow if we see a pattern across the special tests :)